### DISCUSSION TITLE
Double Audio Generation with React Player

### DISCUSSION THREAD

**Discussion by Lakshmi203** [cookpete/react-player#1788] [General]
Description:
I've recently encountered a perplexing issue while utilizing the React Player package for audio playback. It seems that whenever I attempt to play an audio file using the package, the audio is generated twice simultaneously. This results in an undesirable experience for users, as the audio plays with an unintended echo effect.

Steps to Reproduce:

Implement React Player package in a Nextjs application.
Url passing to react player component
Observe that the audio is generated twice, leading to an echo effect.
Expected Behavior:
The React Player package should play the audio file once, producing a single instance of sound.

Actual Behavior:
The audio file is generated twice simultaneously, resulting in a double audio playback effect.

Additional Notes:

This issue persists across different browsers and operating systems.
I have attempted to troubleshoot the problem by reviewing the React Player documentation and exploring potential configuration settings, but to no avail.
It's worth noting that this issue seems to be specific to audio playback, as video playback behaves as expected without any duplication.

**Comment by lludol** [comment]
Hello, I have the same issue! My player is also only for Audio and I have the bug randomly. Sometimes I have it on Chrome other times on Safari (on the phone, tablet, or desktop). I can't manage to find a way to reproduce it every time...

**Comment by emanuelschmitt** [comment]
Hey folks, did you manage to resolve the issue somehow?

**Comment by imjac0b** [comment]
This seems to be happening more often on react 19 with compiler, even with 'use no memo'.

**Comment by calderon** [comment]
It is happening to me, randomly. It seems to happen more often when I use it from a mobile device. Using react-player 2.16.0
